pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/FunctionsBase.java

```diff
@@ -94,14 +183,106 @@ public void registerFunction(final @PathParam("tenant") String tenant,
+                            "name  \n" +
+                            "  The name of functions.  \n" +
+                            "customSchemaInputs  \n" +
+                            "  The map of input topics to Schema class names (The customSchemaInputs is JSON string).  \n" +
+                            "customSerdeInputs  \n" +
+                            "  The map of input topics to SerDe class names (The customSerdeInputs is JSON string).  \n" +
+                            "deadLetterTopic  \n" +
+                            "  Messages that are not processed successfully are sent to `deadLetterTopic`.  \n" +
+                            "runtimeFlags  \n" +
+                            "  Any flags that you want to pass to the runtime. Note that in thread mode, these flags have no impact.  \n" +
+                            "fqfn  \n" +
+                            "  The Fully Qualified Function Name (FQFN) for the function.  \n" +
+                            "inputSpecs  \n" +
+                            "  A generalized way of specifying inputs.  \n" +
+                            "inputs  \n" +
+                            "  The input topic or topics (multiple topics can be specified as a comma-separated list) of functions.  \n" +
+                            "jar  \n" +
+                            "  Path to the JAR file for the function (if the function is written in Java). " +
+                            "  It also supports URL path [http/https/file (file protocol assumes that file " +
+                            "  already exists on worker host)] from which worker can download the package.  \n" +
+                            "py  \n" +
+                            "  Path to the main Python file/Python Wheel file for the function (if the function is written in Python).  \n" +
+                            "go  \n" +
+                            "  Path to the main Go executable binary for the function (if the function is written in Go).  \n" +
+                            "logTopic  \n" +
+                            "  The topic to which the functions logs are produced.  \n" +
+                            "maxMessageRetries  \n" +
+                            "  How many times should we try to process a message before giving up.  \n" +
+                            "output  \n" +
+                            "  The functions output topic (If none is specified, no output is written).  \n" +
+                            "outputSerdeClassName  \n" +
+                            "  The SerDe class to be used for messages output by the function.  \n" +
+                            "parallelism  \n" +
+                            "  The functions parallelism factor (i.e. the number of function instances to run).  \n" +
+                            "processingGuarantees  \n" +
+                            "  The processing guarantees (aka delivery semantics) applied to the function." +
```

Comment:
Don’t assume that a non-native speaker will understand a non-technical abbreviation
whose meaning is obvious to you. Some abbreviations are puzzling to translators as well.

For example, a.k.a. or AKA.